src/logger/loggerApi.c

```diff
@@ -237,14 +237,20 @@ static void get_imu_status(struct Serial *serial, const bool more)
 
 static void get_wifi_status(struct Serial* serial, const bool more)
 {
+        const LoggerConfig *lc = getWorkingLoggerConfig();
```

Comment:
I guess we don't need to #ifdef  wifi status as everything supports it currently. Legit.